test/Kernel.UnitTests/AccessValidator/AccessValidatorTests.cs

```diff
@@ -0,0 +1,221 @@
+﻿using LT.DigitalOffice.Kernel.AccessValidator.Interfaces;
+using LT.DigitalOffice.Kernel.AccessValidator.Requests;
+using LT.DigitalOffice.Kernel.Broker;
+using MassTransit;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.Primitives;
+using Moq;
+using NUnit.Framework;
+using System;
+using System.Collections.Generic;
+using System.Net;
+using System.Threading.Tasks;
+using AV = LT.DigitalOffice.Kernel.AccessValidator;
```

Comment:
You can just write `using LT.DigitalOffice.Kernel.AccessValidator`

Reply:
not work